src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -125,6 +126,12 @@ public string WorkingDirectory
         public bool ErrorDialog { get; set; }
         public IntPtr ErrorDialogParentHandle { get; set; }
 
+        public string UserName
+        {
+            get => _userName ?? string.Empty;
+            set => _userName = value;
```

Comment:
maybe use null for "no name" and throw for setting empty? I am not sure there is a concept of an empty user name

Reply:
This property was copied from the [Windows specific code](https://github.com/dotnet/corefx/blob/3b2666a7aed005c98fcb4f8ca5245c9459be311f/src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Windows.cs#L16-L20).  I don't think it is a good idea to change the behavior.